replace_using_package_version/replace_using_package_version.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
This will break running the the service via poetry in development. Please use a `sed` call in `%build` instead.

Reply:
done